docs/docs/policies.mdx

```diff
@@ -396,6 +398,308 @@ for details.
++---------------------------------------+------------------------+--------------------------------------------------------------+
+| ignore_intents_list                   | []                     | This parameter lets you configure `IntentTEDPolicy` to ignore|
+|                                       |                        | the prediction of `action_unlikely_intent` for a subset of   |
+|                                       |                        | intents. You might want to do this if you come across a      |
+|                                       |                        | certain list of intents for which there are too many false   |
+|                                       |                        | warnings generated.                                          |
++---------------------------------------+------------------------+--------------------------------------------------------------+
+| tolerance                             | 0.0                    | The `tolerance` parameter is a number that ranges from `0.0` |
+|                                       |                        | to `1.0` (inclusive). It helps to adjust the threshold score |
+|                                       |                        | used during prediction of `action_unlikely_intent` at        |
+|                                       |                        | inference time. Here, `0.0` means that the score threshold   |
+|                                       |                        | is the one that `IntentTEDPolicy` had determined at training |
+|                                       |                        | time. A tolerance of `1.0` means that the threshold score    |
+|                                       |                        | is so low that `IntentTED` would not trigger                 |
+|                                       |                        | `action_unlikely_intent` for any of the "negative examples"  |
+|                                       |                        | that it has encountered during training. These negative      |
+|                                       |                        | examples are combinations of dialogue contexts and user      |
+|                                       |                        | intents that are _incorrect_. `IntentTEDPolicy` generates    |
+|                                       |                        | these negative examples from your training data by picking a |
+|                                       |                        | random story part and pairing it with a random intent that   |
+|                                       |                        | doesn't occur at this point.                                 |
++---------------------------------------+------------------------+--------------------------------------------------------------+
+```
+
+</details>
+
+#### Tuning the tolerance parameter
```

Comment:
Should this tuning be done in combination with tuning of the `max_history`?
Especially if the default for max_history is None.

Reply:
Ideally yes, but we haven't experimented with it as of now to recommend how exactly to do this. Maybe we can add a statement on the lines of - "If you change the `max_history` value and retrain a model, you might have to re-adjust the `tolerance` value."
Wdyt?

Reply:
Yes, that would make sense to me.

Reply:
Added it 👍